app/src/main/res/values/strings.xml

```diff
@@ -1093,5 +1093,6 @@
     <string name="no_data_fetched">No Data Fetched</string>
     <string name="recorder">Recorder</string>
     <string name="save_graph">Save Graph</string>
-
+    <string name="expand_all_label">Expand All</string>
```

Comment:
Don't add string resources in the bottom of the file

Reply:
@harsh-2711  i didn't get your point. Where should i put new strings then?

Reply:
Anywhere between the file. Will be good if you place it with the resources being used in that particular file. Keeping it at bottom creates merge conflicts

Reply:
Oh okay. Will make the changes